compiler/optimizer/OMRLocalCSE.cpp

```diff
@@ -1186,6 +1186,9 @@ bool OMR::LocalCSE::canBeAvailable(TR::Node *parent, TR::Node *node, TR_BitVecto
    if (node->getOpCodeValue() == TR::allocationFence)
       return false;
 
+   if (node->getOpCodeValue() == TR::l2a)
```

Comment:
Should `TR::i2a` also be treated the same way (could be used on 32-bit maybe ?) ?

Reply:
If you have time to handle `TR::i2a` in the next day or so, please do so. Otherwise I won't hold up this fix (assuming all tests pass) past tomorrow, since it is addressing a real failure, and you can add the `TR::i2a` in a separate PR.

Reply:
I hadn't touched the other operations that convert to address, as their evaluators don't seem to mark their registers as collected references.  To be safe, I'll capture all of the `*2a` operators in a follow-on pull request that checks the following in `OMR::LocalCSE::canBeAvailable` instead:

```
if (node->getOpCode().isConversion() && node->getOpCode().isRef())
   return false;
```

Reply:
Opened pull request #7218 to address this comment.